ckan/templates/snippets/changes/resource_format.html

```diff
@@ -0,0 +1,37 @@
+<li>
+  <p>
+    {% if ah.method == "add" %}
+      {{ _('Set format of resource <a href="{resource_url}">{resource_name}
+          </a> to <a href="{format_url}">{format}</a> in
+          <a href="{pkg_url}">{dataset}</a>').format(
```

Comment:
It looks like if I use the `{% trans %}` tag instead of `_(`, the formatting of the string to translate might look a little bit nicer, but it still has a bunch of `<a href=...>` tags in. I could take those links out, but I thought that the ability to navigate to the referenced dataset/resource/tag/etc. from the summary of changes would be useful.